pages/design/rule-design.md

```diff
@@ -33,34 +77,74 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input).
+
 ### Authors
```

Comment:
This was updated recently, we're now grouping "authors" under an "acknowledgements". Additionally there is a "previous_authros" field too.

Reply:
Various amendments made to more closely align with ACT Rules Format.